tools/projmgr/src/ProjMgrKernel.cpp

```diff
@@ -52,6 +52,18 @@ XMLTree* ProjMgrKernel::CreateXmlTree() const
 bool ProjMgrKernel::GetInstalledPacks(list<RtePackage*>& packs) {
   list<string> pdscFiles;
   RteFsUtils::GetPackageDescriptionFiles(pdscFiles, theProjMgrKernel->GetCmsisPackRoot(), 3);
+
+  // Find pdsc files from local repository
+  list<string> localPdscUrls;
+  if (!GetLocalPacksUrls(theProjMgrKernel->GetCmsisPackRoot(), localPdscUrls)) {
+    return false;
+  }
+  for (const auto& localPdscUrl : localPdscUrls) {
```

Comment:
Can we add a test case for this situation?

Reply:
Yes, a local pack repository has to be created/emulated.

Reply:
Okay, this test case seems not to be reflected in the code coverage. We might need to check this at some point.